.github/workflows/release.yml

```diff
@@ -35,7 +35,6 @@ jobs:
     strategy:
       matrix:
         node-version: [18.x]
-        pnpm-version: [8.15.7]
```

Comment:
instead of removing this, shall we update the version to `latest` here?

Reply:
Fixed with 31864974f353e49624647150a220104751bc7824